pkg/configuration/configuration_test.go

```diff
@@ -313,6 +313,34 @@ func (s *TestConfigurationSuite) TestIsTestingMode() {
 	})
 }
 
+func (s *TestConfigurationSuite) TestIsE2ETestingMode() {
+	key := configuration.EnvPrefix + "_" + "E2ETESTINGMODE"
+	resetFunc := test.UnsetEnvVarAndRestore(key)
+	defer resetFunc()
+
+	s.Run("default", func() {
+		resetFunc := test.UnsetEnvVarAndRestore(key)
+		defer resetFunc()
```

Comment:
seems that this is redundant ?? even in next test case too.

Reply:
this change is no longer part of this PR: e7b06a11435cad5ff5465e0a8b5c62f842d66d88